Bottom: c4a1105661ee26e1c1716b7f829d333206a1fde9
Top:    90390880827ad75b93a04e8c2346130672fda946
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-04 03:48:29 +0100

Refresh of try-to-avoid-warnings-from

---

diff --git a/src/parse_y.y b/src/parse_y.y
index 6246705..629ccd6 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -164,7 +164,7 @@ parsepcb
 
 				if (!yyPCB)
 				{
-					Message(_("illegal fileformat 1\n"));
+					Message(_("illegal fileformat\n"));
 					YYABORT;
 				}
 				for (i = 0; i < MAX_LAYER + EXTRA_LAYERS; i++)
@@ -252,7 +252,7 @@ parsefont
 
 				if (!yyFont)
 				{
-					Message(_("illegal fileformat 3\n"));
+					Message(_("illegal fileformat\n"));
 					YYABORT;
 				}
 				yyFont->Valid = false;
